projects/valtimo/config/assets/core/de.json

```diff
@@ -54,7 +54,8 @@
       "index": "Index",
       "registrationAt": "Registriert auf",
       "title": "Titel"
-    }
+    },
+    "startSubProcess": "Stárten"
```

Comment:
stárten - typo?

Reply:
It should be fixed now, I used google translate which gave me this version. I asked Sophie and she said it was with just a normal "a"